tests/test_self.py

```diff
@@ -1042,7 +1042,7 @@ def test_regression_parallel_mode_without_filepath(self) -> None:
         path = join(
             HERE, "regrtest_data", "regression_missing_init_3564", "subdirectory/"
         )
-        self._test_output([path, "-j2"], expected_output="No such file or directory")
+        self._test_output([path, "-j2"], expected_output="")
```

Comment:
Isn't this change suspicious ? It seems we might not be testing what we were testing originally.

Reply:
Good Q. I tried to explain in the commit message, but now we have 23 commits :D

This was added as a regression test for a crash. When written it shouldn't have been depending on the expected output being "we can't lint this namespace package". I think it's still okay to keep the test as a regression test for the crash.

Reply:
I didn't rewrite the history but we're close to being able to merge now, do you want me to cleanup ?

Reply:
Not sure I saw the actual comment in the commit, it was mixed with a lot of commit from main, I might have removed it by mistake.